src/rust/ide/src/model/suggestion_database.rs

```diff
@@ -34,6 +37,19 @@ pub struct NoSuchEntry(pub SuggestionId);
 #[fail(display = "Entry named {} does not represent a method.", _0)]
 pub struct NotAMethod(pub String);
 
+#[allow(missing_docs)]
+#[derive(Debug,Fail,Clone)]
+#[fail(display = "Argument index {} is invalid for Entry named {}.",index,name)]
```

Comment:
Entry is capitalized here. Is it referring to some code struct and should have backticks? Or is it just used as the word entry and should not be capitalized?

Reply:
changed to non-capitalized form.